Lib/test/test_subprocess.py

```diff
@@ -1179,7 +1179,7 @@ def test_nonexisting_with_pipes(self):
                 msvcrt.CrtSetReportFile(report_type, msvcrt.CRTDBG_FILE_STDERR)
 
             try:
-                subprocess.Popen(cmd,
+                subprocess.Popen([cmd],
```

Comment:
Do not revert https://github.com/python/cpython/commit/5537646bfacec463b450871dde31cb06c44a0556.  It should be a bugfix no matter what.  cmd is a list and comes through the repr into this python code string as a list.  using [cmd] would make it a list of lists which is never desirable here.